README.md

```
@@ -39,12 +39,9 @@ Install the latest published version from [crates.io](https://crates.io/crates/b
 cargo install blazelint
 ```
 
-> [!TIP]
-> Re-run the command with `--force` to pick up newly published releases.
+Pre-build binaries are available for Linux from the [latest GitHub release](https://github.com/RuztyCrabs/Blazelint/releases/latest). 
 
-Prefer a prebuilt executable? Download the Linux x86_64 binary from the [latest GitHub release](https://github.com/RuztyCrabs/Blazelint/releases/latest) and place it in your `$PATH`.
-
-Windows and MacOS binaries will be added in next release.
+_Windows and MacOS binaries will be added in a later release._
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)